blend4web/build.boot

```diff
@@ -0,0 +1,216 @@
+(def +lib-version+ "16.11")
+(def +version+ (str +lib-version+ "-1"))
+
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all]
+         '[boot.task.built-in :refer :all])
+
+(task-options!
+ pom  {:project     'cljsjs/blend4web
+       :version     +version+
+       :description "Blend4Web -- Javascript WebGL Framework by Triump LLC"
+       :url         "http://www.blend4web.org/"
+       :license     {"GPL3" "https://github.com/TriumphLLC/Blend4Web/blob/master/license/GPL-license.txt"}
+       :scm {:url   "https://github.com/cljsjs/packages"}})
+
+(deftask uranium
+  "Ensure uranium.js and uranium.js.mem are in target directory.  This is not
+  needed to build the package, rather its for any projects using the package."
+  []
+  ;;uranium.js and uranium.js.mem need to both be in the same subdir as b4w's target html.
+  (sift :add-jar {'cljsjs/blend4web #"^*/uranium.js*"}
+        :move {#"^*cljsjs/blend4web/common/uranium.js"
+               "uranium.js"}
+        :to-resource #{#"uranium.js*"}))
+
+(deftask package []
+  (comp
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/deploy/apps/common/b4w.min.js" +lib-version+)
```

Comment:
Huh, instead of downloading all these invidual files, it is much simpler to just download the archive: `https://github.com/TriumphLLC/Blend4Web/archive/16.11.zip`. Just add `:decompress true` option to download task to also extract the files.